client/src/rr/dnssec/keypair.rs

```diff
@@ -241,11 +242,12 @@ impl KeyPair {
     /// # Return
     ///
     /// the KEY record data
+    #[allow(deprecated)]
     pub fn to_sig0key(&self, algorithm: Algorithm) -> DnsSecResult<KEY> {
         self.to_public_bytes()
             .map(|bytes| {
                      KEY::new(Default::default(),
-                              Default::default(),
+                              KeyUsage::Zone,
```

Comment:
BTW, i'm currently struggling to find the RFC deprecating KeyUsage::Zone.
https://tools.ietf.org/html/rfc3445#section-4 still mentions the flag: "The zone bit (bit 7) remains unchanged."
It lists RFCs 4033, 4034, 4035 as obsoleting RFC 3445, but those don't mention KEY RRs, at all. (At least not in their TOCs, didn't look through the full text of the RFCs).

It makes sense that KeyUsage::Zone should not be used, as zone signing is done with DNSKEY RRs, not with KEY RRs. But I'm completely confused by all those cross-references between RFCs. :-)